{{cookiecutter.beat}}/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2016 {{cookiecutter.full_name}}
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Low priority: add a selection of software licenses to choose from:

 "license": ["MIT", "BSD-3", "GNU GPL v3.0", "Apache Software License 2.0"]

and the license file can look like this:
https://github.com/pytest-dev/cookiecutter-pytest-plugin/blob/master/pytest-%7B%7Bcookiecutter.plugin_name%7D%7D/LICENSE